.github/workflows/e2e_test.yaml

```diff
@@ -0,0 +1,188 @@
+name: E2E Tests
+
+on:
+  pull_request:
+
+jobs:
+  e2e-tests-reconcile:
+    runs-on: ubuntu-22.04
+    steps:
+      # Each job has to have this configuration because secrets can be passed through the output of
+      # another job
+      - name: Generate discourse configuration
+        id: configuration
+        run: |
+          echo "discourse= \
+            { \
+              \"hostname\":\"discourse.charmhub.io\", \
+              \"category_id\":\"41\", \
```

Comment:
What is category id `41`, does it make sense to put it in a variable that describes it?

Reply:
Created an issue to track this: https://github.com/canonical/upload-charm-docs/issues/85